README.md

```diff
@@ -5,53 +5,63 @@ transport, supported by [UCX](https://github.com/openucx/ucx#supported-transport
 This open-source project is developed, maintained and supported by [UCF consortium](http://www.ucfconsortium.org/).
 
 ## Runtime requirements
-* Apache Spark 2.2.0/2.3.0/2.4.0
+* Apache Spark 2.3/2.4
 * Java 8+
 * An RDMA-supported network, e.g. RoCE or Infiniband
```

Comment:
maybe best to specify that "UCX library supported transport" and provide link to ucx "supported transports" page
WDYT

Reply:
👍